app/Http/Controllers/TrainingController.php

```diff
@@ -112,6 +114,18 @@ public function store(Request $request)
                 }
             }
 
+            //add credit
+
+            $credit_selected = $request->input('credit');
+            if($credit_selected == "1") {
```

Comment:
Perfect on store!
On update() we have to check if the credit hast change and update (create or delete) it if the change cause that